src/vm/arm64/asmhelpers.asm

```diff
@@ -1326,5 +1329,62 @@ Fail
     LEAF_END
 #endif
 
+;
+; JIT Static access helpers when coreclr host specifies single appdomain flag 
+;
+
+; ------------------------------------------------------------------
+; void* JIT_GetSharedNonGCStaticBase(SIZE_T moduleDomainID, DWORD dwClassDomainID)
+
+    LEAF_ENTRY JIT_GetSharedNonGCStaticBase_SingleAppDomain
+    ; If class is not initialized, bail to C++ helper
+    add x2, x0, #DomainLocalModule__m_pDataBlob
+    ldrb w2, [x2, w1]
```

Comment:
FWIW all test passed with LSL version.  That is probably because for all callers x1 == w1 UXTW.  I will update either way.